go.mod

```diff
@@ -6,8 +6,8 @@ require (
 	github.com/apparentlymart/go-cidr v1.1.0
 	github.com/aws/aws-sdk-go v1.38.23
 	github.com/go-logr/logr v1.2.3
-	github.com/openshift/api v0.0.0-20220525145417-ee5b62754c68
-	github.com/openshift/client-go v0.0.0-20220525160904-9e1acff93e4a
+	github.com/openshift/api v0.0.0-20220929131120-2bde012f248a
```

Comment:
Wouldn't it be `go get -u github.com/openshift/api@2bde012f248a5172dcde2f7104caf0726cf6d93a`?

Reply:
This is in the commit message